### PR TITLE
(docs typo: 'projet' -> 'project')

### PR DESCRIPTION
**Pull request by ms-tg**
Thanks all who ported my sample project here to the new Reactive Streams system. It really makes it a lot easier to understand the changes. Especially thanks for keeping the test working.

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @ms-tg,

Thank you for your contribution! We really value the time you've taken to put this together.

We see that you have signed the Typesafe Contributors License Agreement before, however, the CLA has changed since you last signed it.
Please review the new CLA and sign it before we proceed with reviewing this pull request:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md [hunk 1]
```diff
@@ -119,4 +119,4 @@ For a full example, have a look at [this sample projet](https://github.com/playf
...
 
-The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
\ No newline at end of file
+The consequence of the above is that you are in charge of creating and managing your project's database schema. Therefore, whenever you make a change a Slick table in the code, make sure to also update the database schema. If you find it tedious to manually keep in sync your database schema and the related table definition in your code, you may want to have a look at the code generation feature available in Slick.
```
Comment (dotta) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md]: I don't see any change here. Am I missing something?

**Comment by dotta** [comment]
@ms-tg Not sure why the `typesafe-cla-validator` job is complaining, but I saw you signed the cla so it's all good. If you could have a look at my comment [above](https://github.com/playframework/play-slick/pull/271#discussion-diff-35077063) then I'd be happy to merge it.

File: docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md (hunk 1 above)
Comment (ms-tg) [inline docs/manual/working/scalaGuide/main/sql/slick/PlaySlickMigrationGuide.md]: @dotta no, I don't know why github marked that line as changed. This PR is supposed to be the simplest of typo fixes ('projet' -> 'project'), which is why I did it via the Github edit interface on the web. Does that make sense?

**Comment by typesafehub-validator** [comment]
Hi @ms-tg,

Thank you for your contribution! We really value the time you've taken to put this together.

We see that you have signed the Typesafe Contributors License Agreement before, however, the CLA has changed since you last signed it.
Please review the new CLA and sign it before we proceed with reviewing this pull request:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by ms-tg** [comment]
@dotta, ok fixed it from the command line

**Comment by dotta** [comment]
@ms-tg Thanks!

**Comment by dotta** [comment]
@ms-tg I just realised that I should have asked you to resign the Typesafe CLA. I know you have signed an old version, but I'm required to ask you to sign the latest version or I'll have to revert your commits (which I'd really prefer not to do). Would you mind resigning http://www.typesafe.com/contribute/cla?

Thanks.

**Pull request merged**